doc/_ext/collapsible.py

```diff
@@ -0,0 +1,62 @@
+from docutils import nodes
+from docutils.parsers.rst import Directive
+from docutils.parsers.rst.directives import unchanged
+
+collapsible_id_counter = 0
+
+
+class CollapsibleNode(nodes.General, nodes.Element):
+    pass
+
+
+def visit_collapsible_node(self, node):
+    self.body.append(f"""
+<input type="checkbox" id="{node["id"]}" class="hide">
+<label for="{node["id"]}">
+    <span class="collapsible"><b>{node["header"]}</b></span>
```

Comment:
What is the purpose of this span and its class?

Reply:
Also, please remove bold element. The visual style should be done in the style sheet, not in html markup.